reporter/src/main/kotlin/reporters/freemarker/FreemarkerTemplateProcessor.kt

```diff
@@ -335,9 +353,30 @@ class FreemarkerTemplateProcessor(
         @Suppress("UNUSED") // This function is used in the templates.
         fun filterForUnresolvedVulnerabilities(vulnerabilities: List<Vulnerability>): List<Vulnerability> =
                 vulnerabilities.filterNot { input.resolutionProvider.isResolved(it) }
+
+        /**
+         * Return a flag indicating that issues have been encountered during the run of an advisor with the given
+         * [capability] with at least the given [severity]. This typically means that the report is incomplete;
+         * therefore, it should contain a corresponding warning.
+         */
+        @JvmOverloads
+        fun hasAdvisorIssues(capability: AdvisorCapability, severity: Severity = Severity.ERROR): Boolean =
+            input.ortResult.advisor?.results?.advisorResults.orEmpty()
+                .values
+                .flatten()
+                .filter { capability in it.advisor.capabilities }
+                .any { result -> result.summary.issues.any { it.severity.ordinal >= severity.ordinal } }
```

Comment:
There's no need to compare ordinals, we have defined ordering as of https://github.com/oss-review-toolkit/ort/pull/3956.